eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-nugetfeed.yml

```
@@ -61,53 +67,52 @@ jobs:
           AZCOPY_SPA_CLIENT_SECRET: $(azuresdkpartnerdrops-service-principal-key)
         condition: and(succeeded(), ne(variables['SkipCopyFromPartnerDrops'], 'true'))
 
-      - ${{ if ne(variables['SkipSigning'], 'true') }}:
+      - ${{ if eq(parameters.ShouldSign, 'true') }}:
         - template: pipelines/steps/net-signing.yml@azure-sdk-build-tools
           parameters:
             PackagesPath: ${{ parameters.ArtifactsPath }}
             BuildToolsPath: ${{ parameters.BuildToolsRepoPath }}
 
-      - task: MSBuild@1
-        displayName: 'Upload Symbols'
-        inputs:
-          solution: '${{ parameters.BuildToolsRepoPath }}/tools/symboltool/SymbolUploader.proj'
-          msbuildArguments: >
-            /p:PackagesPath=${{ parameters.ArtifactsPath }}
-            /p:MSPublicSymbolsPAT=$(azuresdk-microsoftpublicsymbols-devops-pat)
-            /p:MSSymbolsPAT=$(azuresdk-microsoft-devops-pat)
-            /p:AzureSDKSymbolsPAT=$(azuresdk-azure-sdk-devops-pat)
-        condition: and(succeeded(), ne(variables['SkipPublishSymbols'], 'true'))
-
-      - task: NuGetToolInstaller@1
-        displayName: 'Use NuGet ${{ parameters.NugetVersion }}'
-        inputs:
-          versionSpec: ${{ parameters.NugetVersion }}
-        condition: and(succeeded(), ne(variables['SkipPublishToNuget'], 'true'))
-
-      - ${{ if ne(parameters.DevOpsFeedID, '') }}:
-        - task: NuGetCommand@2
-          displayName: 'Publish to DevOps Feed'
+      - ${{ if eq(parameters.ShouldPublish, 'true') }}:
```

Same comment as above:

```suggestion
      - ${{ if eq(parameters.ShouldPublish, true) }}:
```